### ISSUE TITLE
Changing values in $header-sizes map in _settings.scss doesn't do anything

### ISSUE THREAD

**Issue by designerdean** [zurb/foundation-ssg#5]
When I try to change the $header-sizes map in the _settings.scss file, the header sizes don't change. 

```sass
$header-sizes: (
  small: (
    'h1': 24,
    'h2': 20,
    'h3': 19,
    'h4': 18,
    'h5': 17,
    'h6': 16,
  ),
  medium: (
    'h1': 48,
    'h2': 40,
    'h3': 31,
    'h4': 25,
    'h5': 20,
    'h6': 16,
  )
);
```

It seems odd that "small" and "medium" are the options—is that intentional? Any help is much appreciated.

**Comment by rafibomb** [comment]
Sorry can you re-open this issue here: https://github.com/zurb/foundation-sites-6
It's the place for general framework issues.

**Comment by designerdean** [comment]
The bug doesn't seem to exist there, it's only in the Static Site
Generator's gulpfile.


On Fri, Nov 6, 2015 at 2:08 PM Rafi <notifications@github.com> wrote:

> Closed #5 <https://github.com/zurb/foundation-ssg/issues/5>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zurb/foundation-ssg/issues/5#event-457395112>.
>
-- 
Josh Dean
